CHANGELOG.md

```diff
@@ -1,3 +1,41 @@
+## 1.15.0
+
+This relase brings some expanded DNS configuration options, beta support for QEMU, recursive volume mounts, and rkt-monitor improvements.
+
+### New features
+- DNS configuration improvements ([#3161](https://github.com/coreos/rkt/pull/3161)):
+    - Respect DNS results from CNI
+    - Add --dns=host mode to bind-mount the host's /etc/resolv.conf
+    - Add --dns=none mode to ignore CNI DNS
+    - Add --hosts-entry (IP=HOSTNAME) to tweak the pod's /etc/hosts
+    - Add --hosts-entry=host to bind-mount the host's /etc/hosts
+- Introduce QEMU support as an alternative KVM hypervisor ([#2952](https://github.com/coreos/rkt/pull/2952))
+- rkt-monitor: bunch of improvements ([#3093](https://github.com/coreos/rkt/pull/3093))
+- add support for recursive volume/mounts ([#2880](https://github.com/coreos/rkt/pull/2880))
+- stage1: allow systemd notifications from the app in the container to the host ([#2826](https://github.com/coreos/rkt/pull/2826))
+
+### Other changes
+- makefile/kvm: add dependency for copied files ([#3197](https://github.com/coreos/rkt/pull/3197))
+- store: refactor GetRemote ([#2975](https://github.com/coreos/rkt/pull/2975)).
```

Comment:
IMO such change could be simply skipped in changelog. It does not have an impact on anything from users side like from dev side.